a8cteam51/special-projects-blocks-monorepo
# Add terms list block

### What
This block renders a list of all the terms associated with a user-selected taxonomy.

### Why

Displaying a list of terms from a custom taxonomy is a common requirement for websites and is not currently supported in core.

### How

It reimplements the [Terms List Block](https://github.com/mattyza/terms-list-block/) based on the most recent version of the core `core/categories` block.

### How to Test

Add a Terms List block to any post or template and test to see if you can select different taxonomies, and that all of the block settings (display as dropdown, show post counts, show empty terms, etc.) all work as expected.

https://github.com/user-attachments/assets/130fa9f0-2a43-4abf-9159-965cd8037b2b
